### PR TITLE
Hotfix/v1.0.1: Hotfix in Documentation

### PR DESCRIPTION
**Pull request by waji46**
Title : Hotfix in Documentation

	[PS-1]
	 - Updated startup script
	 - Updated the README file
	 - Updated the Architecture Diagram in README file
	 - Updated the ROADMAP file
	 - Fixed SSL certificate validation issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,30 +79,31 @@ Check the below video for a quick demo of XC3.
...
     git clone https://github.com/XgridInc/xc3.git
     ```
-2. Install the `prometheus client` library using following commands:
-    ```bash
+2. You have to configure the config.sh file in xc3/infrastructure and run the below command
+    ```
+    cd xc3/infrastructure
+    bash init.sh
+    ```
+3. Install the `prometheus client` library using following commands:
```
Comment (muhdsaad) [inline README.md:94]:
```suggestion
3. Install the `prometheus-client` library using following commands:
```

**Review by muhdsaad — CHANGES_REQUESTED** [review]
Update the PR with updated suggestion

**Review by yaseenzafar123 — APPROVED** [review]
LGTM

**Review by muhdsaad — APPROVED** [review]
LGTM

**Pull request merged**